srcpkgs/zig/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'zig'
 pkgname=zig
 version=0.8.0
-revision=1
+revision=2
```

Comment:
`zig` package fixes should be a separate commit.

Reply:
Unless I'm missing something, this wasn't resolved?

Reply:
This was resolved for a while but then I backport a new fix that unfortunately didn't make it into 0.8.1 and forgot about this review comment. I'll split up the commits shortly.